app/assets/stylesheets/views/_taxons.scss

```diff
@@ -23,13 +21,20 @@
 
   h3 {
     @include bold-19;
-    padding-bottom: $gutter-one-third;
+    @include media (tablet) {
+      padding-bottom: $gutter-one-third;
+    }
   }
 
   p {
     @include core-19;
-    margin-top: 2px;
-    margin-bottom: 6px;
+    margin-top: #{(5 / 16)}em;
+    margin-bottom: #{(20 / 16)}em;
+
+    @include media(tablet) {
+      margin-top: #{(5 / 19)}em;
+      margin-bottom: #{(20 / 19)}em;
```

Comment:
@suzannehamilton the type size is 19px, so 20/19em will give you a 20px margin (and we use multiples of 5px for vertical rhythm). I'm not sure why we are using ems though.